js/level.js

```diff
@@ -1,26 +1,51 @@
+let f1 = new Fruit();
+let f2 = new Fruit();
+let f3 = new Fruit();
+let f4 = new Fruit();
+let f5 = new Fruit();
+let f6 = new Fruit();
+let f7 = new Fruit();
+let f8 = new Fruit();
+let f9 = new Fruit();
+let f10 = new Fruit();
+let f11 = new Fruit();
+let f12 = new Fruit();
```

Comment:
Can we dynamically create fruits at each level? 
We are adding a new class for each fruit to be rendered out